src/__tests__/final/06.js

```diff
@@ -52,3 +52,37 @@ test('displays the users current location', async () => {
     `Longitude: ${fakePosition.coords.longitude}`,
   )
 })
+
+test('displays error message when geolocation is not supported', async () => {
+  const fakePosition = {
+    coords: {
+      latitude: 59,
+      longitude: 18,
+    },
+  }
```

Comment:
Because we're not going to resolve the promise, we don't need the `fakePosition` variable in this test :)

```suggestion
```